### PR TITLE
Migrate `service` package and `engine.plugin.conditions`

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: packages/engine/env/centos7/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM centos:7.8.2003
```
Comment (wdavidw) [inline packages/engine/env/centos7/Dockerfile:1]: no

File: packages/engine/src/plugins/conditions_os.coffee [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    conditions.if_os = [conditions.if_os] unless Array.isArray conditions.if_os
+    for condition in conditions.if_os
+      # Normalize conditions
+      condition.name ?= []
+      condition.name = [condition.name] unless Array.isArray condition.name
+      condition.version ?= []
+      condition.version = [condition.version] unless Array.isArray condition.version
+      condition.version = utils.semver.sanitize condition.version, 'x'
+      condition.arch ?= []
+      condition.arch = [condition.arch] unless Array.isArray condition.arch
```
Comment (wdavidw) [inline packages/engine/src/plugins/conditions_os.coffee:38]: not the place to normalize, must be inside the hook `nikita:session:normalize`.

File: packages/engine/src/plugins/conditions_os.coffee [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    conditions.unless_os = [conditions.unless_os] unless Array.isArray conditions.unless_os
+    for condition in conditions.unless_os
+      # Normalize conditions
+      condition.name ?= []
+      condition.name = [condition.name] unless Array.isArray condition.name
+      condition.version ?= []
+      condition.version = [condition.version] unless Array.isArray condition.version
+      condition.version = utils.semver.sanitize condition.version, 'x'
+      condition.arch ?= []
+      condition.arch = [condition.arch] unless Array.isArray condition.arch
```
Comment (wdavidw) [inline packages/engine/src/plugins/conditions_os.coffee:81]: nikita:session:normalize

File: packages/engine/env/centos7/Dockerfile (hunk 1 above)
Comment (sergkudinov) [reply packages/engine/env/centos7/Dockerfile]: fixed

**Pull request merged**